libobs-opengl/gl-cocoa.m

```diff
@@ -419,9 +419,8 @@ bool gs_texture_rebind_iosurface(gs_texture_t *texture, void *iosurf)
 		blog(LOG_ERROR, "Unexpected pixel format: %d (%c%c%c%c)", pf,
 		     pf >> 24, pf >> 16, pf >> 8, pf);
 
-	if (tex->width != IOSurfaceGetWidth(ref) ||
-	    tex->height != IOSurfaceGetHeight(ref))
-		return false;
+	tex->width = IOSurfaceGetWidth(ref);
+	tex->height = IOSurfaceGetHeight(ref);
```

Comment:
Is this directly related to SCK or does it fix an issue only exposed by SCK?
I don't know the direct effects of this, but to me it feels like it should be split into a separate commit with a small explanation of what it does.

Reply:
I think this allows the source to scale with the size of the captured window - that is, if the window increases in size, the source automagically matches it.